### ISSUE TITLE
Craft Preview - Neon block children are missing

### ISSUE THREAD

**Issue by r0skar** [spicywebau/craft-neo#458]
#### Description
When using the internal preview tool (preview target is on a different domain and fetches data via the GraphQL endpoint) and/or the GraphQL explorer, `children` are always returned as an empty array.

#### Steps to reproduce
Query a Neo block interface with children:

```gql
query Test {
  entry(draftId: 438) {
    ... on dataCard_dataCard_Entry {
      __typename
      uid
      sourceId
      cardData: dataCardData {
        ... on dataCardData_graph_BlockType {
          label: dataCardDataLabel
          charts: children {
            ... on dataCardData_charts_BlockType {
              label: dataCardChartLabel
            }
          }
        }
      }
    }
  }
}

```

The result returns an empty array for the `charts` children, although the data exists and can be queried in live mode.

```json
{
  "data": {
    "entry": {
      "__typename": "dataCard_dataCard_Entry",
      "uid": "20ab616e-3963-41bb-837c-13e1689c2c74",
      "sourceId": 20631,
      "cardData": [
        {
          "label": "Total Workforce",
          "charts": []
        }
      ]
    }
  }
}
```

#### Other information
- Neo version: 2.9.9
- Craft version: Craft Pro 3.6.14

**Comment by ttempleton** [comment]
Seems like this happens when either the `draftId` or `revisionId` are given to the `entry` query.  Will look into how we can fix this.

**Comment by ttempleton** [comment]
This should be fixed now in Neo 2.9.10.
